.github/settings.yml

```diff
@@ -1,28 +1,6 @@
-# The settings are inherited from common repository repo-settings (same file name and location)
-# You can append settings that are deeply merged with the inherited settings.
-#
-# When you have installed the GitHub App "repo-settings" in this repository,
-# any change of this settings.yml file is detected by the GitHub App and
-# the settings of this repository are updated immediately.
-#
-_extends: repo-settings:.github/common-settings-v2.yml
+_extends: repo-settings:.github/iac-onepipeline-settings.yaml
```

Comment:
this is wrong - put it back to `_extends: repo-settings:.github/common-settings-v2.yml`